core/src/main/scala/ai/h2o/sparkling/backend/converters/TimeZoneConversions.scala

```diff
@@ -25,15 +25,15 @@ import org.apache.spark.sql.catalyst.util.DateTimeUtils
 trait TimeZoneConversions {
   def sparkTimeZone: TimeZone
 
-  def fromSparkTimeZoneToUTC(timestamp: Long): Long = DateTimeUtils.fromUTCTime(timestamp, sparkTimeZone.getID)
+  def fromSparkTimeZoneToUTC(timestamp: Long): Long = DateTimeUtils.toUTCTime(timestamp, sparkTimeZone.getID)
```

Comment:
I believe we were using opposite methods in this class

Reply:
This was made like this on purpose. It seems that Spark internal methods has opposite meaning.

Reply:
The python tests I added should show us the correct behavior